packages/workbox-core/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "workbox-core",
+  "version": "1.2.0",
+  "license": "Apache-2.0",
+  "author": "Google's Web DevRel Team",
+  "description": "This module is used by a number of the other workboxjs.org modules to share common code.",
+  "repository": "googlechrome/workbox",
+  "bugs": "https://github.com/googlechrome/workbox/issues",
+  "homepage": "https://github.com/GoogleChrome/workbox/tree/master/packages/workbox-core",
+  "keywords": [
+    "workbox",
+    "workboxjs",
+    "service worker",
+    "sw"
+  ],
+  "scripts": {
+    "pretest": "node ../../infra/build-test-bundle.js",
+    "test": "mocha './test/bundle-build/**/*.js'"
```

Comment:
Originally was trying to set this up without gulp, but it was going to be to restrictive (at least I think it would be in the long run). Switching to Gulp 4 was the middle ground IMO. Will Remove.